lib/python/Screens/FlashManager.py

```diff
@@ -0,0 +1,600 @@
+			<convert type="ConditionalShowHide" />
+		</widget>
+	</screen>"""
+
+	def __init__(self, session):
+		Screen.__init__(self, session)
+		HelpableScreen.__init__(self)
+		self.skinName = ["FlashManager", "FlashOnline"]
+		self.imageFeed = "OpenATV"
+		self.setTitle(_("Flash Manager - %s Images") % self.imageFeed)
+		self.imagesList = {}
+		self.expanded = []
+		self.setIndex = 0
+		self["actions"] = HelpableActionMap(self, ["OkCancelActions", "ColorActions", "NavigationActions"], {
+			"cancel": (self.keyCancel, _("Cancel the image selection and exit")),
+			"close": (self.closeRecursive, _("Cancel the image selection and exit all menus")),
+			"ok": (self.keyOk, _("Select the highlighted image and proceed to the slot selection")),
+			"red": (self.keyCancel, _("Cancel the image selection and exit")),
+			"green": (self.keyOk, _("Select the highlighted image and proceed to the slot selection")),
+			"yellow": (self.keyDistribution, _("Select a distribution from where images are to be obtained")),
+			"blue": (self.keyDeleteImage, _("Delete the selected locally stored image")),
+			"top": (self.keyTop, _("Move to first line / screen ")),
+			"pageUp": (self.keyPageUp, _("Move up a screen")),
+			"up": (self.keyUp, _("Move up a line")),
+			"down": (self.keyDown, _("Move down a line")),
+			"pageDown": (self.keyPageDown, _("Move down a screen")),
+			"bottom": (self.keyBottom, _("Move to last line / screen"))
+		}, prio=-1)
+		self["key_red"] = StaticText(_("Cancel"))
+		self["key_green"] = StaticText()
+		self["key_yellow"] = StaticText(_("Distribution"))
+		self["key_blue"] = StaticText()
+		self["description"] = StaticText()
+		self["list"] = ChoiceList(list=[ChoiceEntryComponent("", ((_("Retrieving image list - Please wait...")), "Loading"))])
+		self.callLater(self.getImagesList)
+
+	def getImagesList(self):
+		def getImages(path, files):
+			for file in [x for x in files if splitext(x)[1] == ".zip" and self.box in x]:
+				try:
+					if checkImageFiles([x.split(sep)[-1] for x in ZipFile(file).namelist()]):
+						imageType = _("Downloaded images")
+						if "backup" in file.split(sep)[-1]:
+							imageType = _("Backup images")
+						if imageType not in self.imagesList:
+							self.imagesList[imageType] = {}
+						self.imagesList[imageType][file] = {
+							"link": str(file),
+							"name": str(file.split(sep)[-1])
+						}
+				except Exception:
+					print("[FlashManager] Error: Unable to extract file list from Zip file '%s'!" % file)
+
+		if not self.imagesList:
+			try:
+				feedURL, boxInfoField = FEED_URLS.get(self.imageFeed, ("http://images.mynonpublic.com/openatv/json/%s", "machinebuild"))
+				self.box = BoxInfo.getItem(boxInfoField, "")
+				url = feedURL % self.box
+				req = Request(url, None, {'User-agent' : 'Mozilla/5.0 (Windows; U; Windows NT 5.1; de; rv:1.9.1.5) Gecko/20091102 Firefox/3.5.5'})
```

Comment:
Is this URL requesting a German language version of the page?  It should be requesting US English or English at least.

I would also use double quotation marks and correct the spacing before the colon:
req = Request(url, None, {"User-agent": "Mozilla/5.0 (Windows; U; Windows NT 5.1; de; rv:1.9.1.5) Gecko/20091102 Firefox/3.5.5"})